Kernel/FileSystem/FUSE/FileSystem.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2024, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/IntegralMath.h>
+#include <Kernel/Devices/DeviceManagement.h>
+#include <Kernel/FileSystem/FUSE/FileSystem.h>
+#include <Kernel/FileSystem/FUSE/Inode.h>
+#include <Kernel/Tasks/Process.h>
+#include <Kernel/Time/TimeManagement.h>
+
+namespace Kernel {
+
+struct [[gnu::packed]] FUSESpecificFlagsBytes {
+    u64 pid;
+    u64 device_inode;
+    u64 rootmode;
+    u64 gid;
+    u64 uid;
+};
+
+static u64 reinterpret_as_octal(u64 decimal)
```

Comment:
Could we move this to a more standardized place? something like AK? seems like this could be easily be in our math section of the AK library.